src/entry.rs

```diff
@@ -309,6 +320,29 @@ impl<'a> EntryFields<'a> {
                                         for `{}`", mode, dst.display()), e)
             }));
         }
+        // Windows does not completely support posix xattrs
+        // https://en.wikipedia.org/wiki/Extended_file_attributes#Windows_NT
+        if cfg!(target_family = "unix") && preserve_xattrs {
```

Comment:
By default, gnu tar does not preserve extended attributes. You need to specify "--xattrs" option.
If we want to preserve xattrs by default, we probably need to ignore any
errors while setting extended attributes, if we do not want to prohibit the use of unpack() on tmpfs or other virtual fs*.

Should I delete functions unpack_preserving_xattrs() and move processing xattrs in unpack(), ignoring xattrs::set() errors?

* As I understand from the documentation, linux does not support user_xattr for tmpfs in vanilla kernel. Only  ACLs/capabilities/other system attributes. Perhaps, ignoring errors of setting these attributes is the right decision.